src/components/i18n/i18n.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import i18n from 'i18n-js';
+import I18nComponent from './i18n.js';
+
+<Meta title="i18nComponent" parameters={{ info: { disable: true }}} />
+
+# I18nComponent (Internationalization)
+Displays a translation from an i18n file.
+
+Internationalization (i18n) means apps can be localized for languages and cultures easily.
+This component displays a single text translation.
+You can use this component with any other component that displays text.
+
```

Comment:
I think we should highlight that we don't reccomend using this component any more.

Please can you add a warning using the `DeprecationWarning` component.

Something along the lines of 

> It's on our roadmap to remove our I18n implementation and provide localisation for our components out of the box. If you're adding localisation to your application we reccomend you use a dedicated solution such as `react-i18next`

Reply:
I've added the deprecation warning as requested 😄